### PR TITLE
Update Node Imports

### PR DESCRIPTION
**Pull request by sybohy**

### CODE REVIEW

**Review by seveibar — APPROVED** [review]

File: docs/thermostats/get-thermostat.md [hunk 1]
```diff
@@ -69,8 +69,8 @@ print(thermostat)
 {% endtab %}
 
 {% tab title="Javascript" %}
-```typescript
-import Seam from "seamapi"
+```javascript
+import Seam from "se"
```
Comment (sybohy) [inline docs/thermostats/get-thermostat.md:73]: this is wrong

File: snippet-playground/javascript/main.mjs [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-import Seam from "seamapi"
+import Seam from "seam"
```
Comment (DebbieAtSeam) [inline snippet-playground/javascript/main.mjs:3]:
```suggestion
import Seam from "seamapi"
```

**Review by DebbieAtSeam — CHANGES_REQUESTED** [review]

**Review by DebbieAtSeam — APPROVED** [review]

File: snippet-playground/javascript/main.mjs (hunk 2 above)
Comment (sybohy) [reply snippet-playground/javascript/main.mjs]: oof. Thank you! This is why we do code reviews 😄

File: docs/core-concepts/authentication/api-keys.md [hunk 3]
```diff
@@ -74,7 +75,7 @@ console.log(workspace);
   is_sandbox: true
 }
 */
-</code></pre>
+
```
Comment (sybohy) [inline docs/core-concepts/authentication/api-keys.md:78]: This is missing code block closing  backticks

**Review by sybohy — COMMENTED** [review]
Not ready for merge

**Pull request merged**